docs/development/profiler.md

```diff
@@ -0,0 +1,55 @@
+## Profiler (Experimental)
+
+Currently, DJL supports experimental profilers for developers that 
+investigate the performance of operator execution as well as memory consumption.
+As we are still working in progress on the feature, different engines have different APIs and produce different output format.
+In the future, we are considering to design a unified APIs and output unified format. 
+
+### MXNet
+
+By setting the following environment variable, it generates `profile.json` after executing the code.
+```
+export MXNET_PROFILER_AUTOSTART=1
+```
+You can view it in a browser using trace consumer like `chrome://tracing `. Here is a snapshot that shows the sample output.
+![img](https://raw.githubusercontent.com/dmlc/web-data/master/mxnet/tutorials/python/profiler/profiler_output_chrome.png)
+
+### PyTorch
+
+DJL have integrated PyTorch C++ profiler API and expose `JniUtils.startProfile` and `JniUtils.stopProfile(outputFile)` Java APIs.
+`JniUtils.startProfile` takes `useCuda(boolean)`, `recordShape(boolean)` and `profileMemory(boolean)` arguments respectively.
+`useCuda` indicates if profiler enables timing of CUDA events using the cudaEvent API.
+`recordShape` indicates if information about input dimensions will be collected or not.
+`profileMemory` indicates if profiler report memory usage or not.
+`JniUtils.stopProfile` takes a outputFile of String type.
+
+Wrap the code snippet you want to profile in between `JniUtils.startProfile` and `JniUtils.stopProfile`.
+Here is an example
+```
```

Comment:
I think you need empty lines before and after code blocks. Consecutive lines in markdown means sentences in the same paragraph but you can't have a code block in a paragraph. Github could handle it, but mkdocs had problems